amplipi/zeroconf.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python3
+
+# AmpliPi Home Audio
+# Copyright (C) 2022 MicroNova LLC
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""AmpliPi zeroconf service advertisement"""
+
+import logging
+from multiprocessing.synchronize import Event as SyncEvent
+from socket import gethostname, inet_aton
+import sys
+from time import sleep
+from typing import Callable, Optional, Tuple, Union
+
+import netifaces as ni
+from zeroconf import IPVersion, ServiceInfo, Zeroconf
+
+from amplipi import utils
+
+# TEST: use logging in this module
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
```

Comment:
nitpick: I think about the only thing I might change about the logging would be to either:
1) not set the log level at all. this defaults to `logging.WARNING`, _but_ permits us to set the log level elsewhere/at the root logger (notably during app startup). Should we fully migrate to ussing `logging` at some point, we'll probably poke this then, which is why this is a nitpick, OR
2) use an environment variable to expose this straight away and set a default to `logging.INFO` when it is unset. I like this less, but gives us some flexibility straight away.

personally? I'd probably just leave this as-is til we migrate all `print` statements.